.github/workflows/run-pr-tests.yaml

```diff
@@ -1,4 +1,4 @@
-name: PR build
+name: Run tests in a branch
```

Comment:
@DementevNikita, is it possible to give a shorter name?
(For instance: `Test`)

So that names of jobs won't be hidden because of the long workflow name: https://ibb.co/MG7mxvh

Reply:
renamed to `Run tests`